WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -1962,8 +1962,6 @@ static GSourceFuncs _handlerIntervention =
         {
             _adminLock.Lock();
 
-            _URL = URL;
```

Comment:
This will make URL property query return stale values. And it is not clear why it is removed. So, I'd prefer not to submit this change.

Reply:
When curl command with Invalid URL param is issued, then LoadFailed log is displaying for both invalid and valid URL's  i.e, https://www.tdk-page-test.com/ (invalid) and https://www.google.com/(valid).

Code flow:

Here, in (*index)->LoadFailed(_URL); (https://github.com/rdkcentral/rdkservices/blob/8cbee6bc86dfd911485ca63365798996ecee816c/WebKitBrowser/WebKitImplementation.cpp#L2026) function call, _URL is taking previous loaded URL and showing LoadFailed for previously loaded URL instead of current invalid URL.

Observed _URL is updated in OnURLChanged() method as shown (_URL = URL;)
https://github.com/rdkcentral/rdkservices/blob/8cbee6bc86dfd911485ca63365798996ecee816c/WebKitBrowser/WebKitImplementation.cpp#L1965

And also _URL is used only to pass as argument for LoadFailed (https://github.com/rdkcentral/rdkservices/blob/8cbee6bc86dfd911485ca63365798996ecee816c/WebKitBrowser/WebKitImplementation.cpp#L2026). Nowhere it is being used.

So commented the line to resolve the logging issue.